src/App.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <Header />
+  <!-- <div id="nav">
+    <router-link to="/">Home</router-link> |
+  </div> -->
+  <main>
+    <router-view/>
+  </main>
+  <Footer />
+</template>
+
+<script>
+import Header from '@/components/Header/index.vue';
+import Footer from '@/components/Footer/index.vue';
+
+export default {
+  name: 'Home',
+  components: {
+    Header,
+    Footer,
+  },
+};
+</script>
+
+<style>
+#app {
+  font-family: Avenir, Helvetica, Arial, sans-serif;
```

Comment:
All the resettings should be there in the style.css file as `*{box-sizing: border-box; margin: 0; padding: 0}` and set font-family inside body / html tag.

Reply:
Yea, that maybe the case for views, but definitely not for components.
That's subjective tho. We can have a separate discussion on this :)
.
Resolving this one for now.